packages/paste-core/primitives/box/src/index.tsx

```diff
@@ -67,6 +67,7 @@ interface BaseBoxProps
   backgroundAttachment?: BackgroundAttachmentProperty;
   outline?: OutlineProperty<string>;
   float?: FloatProperty;
+  htmlFor?: string;
```

Comment:
We don't need this anymore I don't think

Reply:
Right. Removed.